ServiceTier/Services/PresetService.cs

```diff
@@ -57,15 +58,14 @@ public PresetDTO GetPresetById(int presetid)
                 throw new PresetNotFoundException();
             }
             var preset = unitOfWork.Presets.GetByID(presetid);
-
             return preset.ToPresetDTO(timerService.GetAllTimersByPresetId(presetid));
         }
 
         public List<PresetDTO> GetAllCustomPresetsByUserId(int userid)
         {
             var listOfPresetsDTO = new List<PresetDTO>();
-            var presets = unitOfWork.Presets.GetAll();
-            var timers = unitOfWork.Timers.GetAll();
+            var presets = unitOfWork.Presets.GetAllEntities();
```

Comment:
and why not to use GetAllEntitiesByFilter here?

Reply:
I did change other methods of Preset service to ones using GetAllEntities() method of the Repository now returning IEnumerable<> for the application to work. But I didn't want to do the same for the cases where GetAllEntitiesByFilter() should have been applied all in the same pull request. I or @oleksandrlytvyn1223 are going to do it in a separate pull request, if it's ok. As for other issues with styling/formatting - I'm going to refactor my code in my next pull request regarding the unit tests for Tasks controller.